docs/resources/vpc_subnet_v1.md

```diff
@@ -78,18 +86,20 @@ The following arguments are supported:
 
 ## Attributes Reference
 
-All of the argument attributes are also exported as result attributes:
+All the argument attributes are also exported as result attributes:
 
-* `id` - Specifies a resource ID in UUID format. Same as OpenStack network ID (`OS_NETWORK_ID`).
+* `id` - Specifies a resource ID in UUID format. Same as OpenStack Neutron Network ID (`OS_NETWORK_ID`).
```

Comment:
Saying `OpenStack Neutron Network` is overkill, either `OpenStack network` or `Neutron network`